AlphaWallet/Transfer/Controllers/TransactionConfigurator.swift

```diff
@@ -202,24 +210,38 @@ class TransactionConfigurator {
     }
 
 // swiftlint:disable function_body_length
-    private static func createConfiguration(server: RPCServer, transaction: UnconfirmedTransaction, account: AlphaWallet.Address) -> TransactionConfiguration {
+    private static func createConfiguration(server: RPCServer, transaction: UnconfirmedTransaction, account: AlphaWallet.Address) throws -> TransactionConfiguration {
```

Comment:
So the main changes are in this function and the rest is just propogating the error/throw as well as displaying the error, right?

Reply:
Correct